lib/analyzer/apk-analyzer.ts

```diff
@@ -1,26 +1,28 @@
 import { AnalyzerPkg } from "./types";
 export { analyze };
+import { Docker } from "../docker";
 
-const APK_DB_INSTALLED = "lib/apk/db/installed";
+const APK_DB_INSTALLED = "/lib/apk/db/installed";
 
 const APK_PKGPATHS = [APK_DB_INSTALLED];
 
 export { APK_PKGPATHS };
 
-async function analyze(
-  targetImage: string,
-  pkgFiles: { [key: string]: string },
-) {
-  const dbFile = pkgFiles[APK_DB_INSTALLED];
-  const pkgs = dbFile ? parseFile(dbFile) : [];
+async function analyze(docker: Docker) {
+  const pkgs = await getPackages(docker);
   return {
-    Image: targetImage,
+    Image: docker.getTargetImage(),
     AnalyzeType: "Apk",
     Analysis: pkgs,
   };
 }
 
-function parseFile(text: string) {
+async function getPackages(docker: Docker): Promise<AnalyzerPkg[]> {
+  const dbFileContent = await docker.getFileProduct(APK_DB_INSTALLED);
```

Comment:
can `getFileProduct` return undefined? if so, you don't need to call `parseFile`